elm.json

```
@@ -19,10 +19,11 @@
         "elm-community/list-extra": "8.0.0 <= v < 9.0.0",
         "folkertdev/elm-deque": "3.0.0 <= v < 4.0.0",
         "folkertdev/svg-path-lowlevel": "3.0.0 <= v < 4.0.0",
-        "ianmackenzie/elm-geometry": "1.1.0 <= v < 2.0.0"
+        "ianmackenzie/elm-geometry": "3.3.0 <= v < 4.0.0",
+        "ianmackenzie/elm-units": "2.3.0 <= v < 3.0.0"
```

```suggestion
        "ianmackenzie/elm-geometry": "3.0.0 <= v < 4.0.0",
        "ianmackenzie/elm-units": "2.0.0 <= v < 3.0.0"
```

Can these ranges be safely expanded? It's best to have these as wide as possible to avoid dependency difficulties further down.